.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  # Pub dependencies
+  - package-ecosystem: "pub"
+    directories:
+      - packages/**/*
+      - modules/**/*
+      - integrations/**/*
+      - templates/serverpod_templates
+      - tools/serverpod_cli
+    schedule:
+      interval: "daily"
+    versioning-strategy: "widen"
+    # Since we are using a monorepo, we want to update the same package
+    # in all directories at the same time.
+    # Currently dependabot does not support this, so we limit the number of
+    # open pull requests to 1 to avoid conflicts.
```

Comment:
Perhaps you could link to the issue you created here as well?

Reply:
There is no specific issue so far, I have opened a discussion question to try and figure out if it is possible to achieve in some way.